tests/integration/tables/keychain_items.cpp

```diff
@@ -23,7 +23,7 @@ class KeychainItemsTest : public testing::Test {
 
 TEST_F(KeychainItemsTest, test_sanity) {
   auto const data = execute_query("select * from keychain_items");
-  ASSERT_GT(data.size(), 0ul);
+  ASSERT_GT(data.size(), 1ul);
```

Comment:
Why the change here?

Reply:
User error. This should be testing for not-empty, and I'm used to `GE 1ul` and not `GT 0ul`